src/common/pre-deploy-phase-common.ts

```diff
@@ -17,11 +17,12 @@
 import * as winston from 'winston';
 import * as cloudformationCalls from '../aws/cloudformation-calls';
 import * as ec2Calls from '../aws/ec2-calls';
-import { AccountConfig, PreDeployContext, ServiceConfig, ServiceContext } from '../datatypes';
+import {AccountConfig, PreDeployContext, ServiceConfig, ServiceContext, Tags} from '../datatypes';
 import * as deployPhaseCommon from './deploy-phase-common';
 import * as handlebarsUtils from './handlebars-utils';
+import {getTags} from './tagging-common';
 
-async function createSecurityGroupForService(stackName: string, sshBastionIngressPort: number | null, accountConfig: AccountConfig) {
+async function createSecurityGroupForService(stackName: string, sshBastionIngressPort: number | null, accountConfig: AccountConfig, tags: Tags) {
```

Comment:
Wow thanks for catching this, I didn't even realize our security group stacks weren't being tagged!

Reply:
You can thank IntelliJ's Find Usages inspection :grin: I just ran a find usage on the create stack call and went from there.